test/TesterAzureUtils/Streaming/HaloStreamSubscribeTests.cs

```diff
@@ -63,18 +67,19 @@ public HaloStreamSubscribeTests(Fixture fixture)
         {
             this.fixture = fixture;
             HostedCluster = fixture.HostedCluster;
+            fixture.EnsurePreconditionsMet();
         }
-        
+
         public void Dispose()
         {
-            var deploymentId = this.HostedCluster.DeploymentId;
+            var deploymentId = this.HostedCluster?.DeploymentId;
             if (_streamProvider != null && _streamProvider.Equals(AzureQueueStreamProviderName))
```

Comment:
Above you check that deploymentId isn't null, but not here.  Should the check be NullOrWhitespace?